examples/nodejs/say.ts

```diff
@@ -1,13 +0,0 @@
-import Client, {connect} from "@dagger.io/dagger"
```

Comment:
Just noticed this one isn't just an extension, so do we want to keep it? Or has it been replaced by docs?

Reply:
we'll rescue this code and move to `dagger/examples`.